src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -135,6 +136,40 @@ public partial class Window
             }
         }
 
+        /// <summary>
+        /// An event for the touched signal which can be used to subscribe or unsubscribe the event handler provided by the user.<br />
+        /// The touched signal is emitted when the touch input is received.<br />
+        /// This can receive touch events before child. <br />
+        /// If it returns false, the child can receive the touch event. If it returns true, the touch event is intercepted. So child cannot receive touch event.<br />
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event ReturnTypeEventHandler<object, TouchEventArgs, bool> InterceptTouchEvent
+        {
+            add
+            {
+                if (rootLayerInterceptTouchDataEventHandler == null)
+                {
+                    rootLayerInterceptTouchDataCallback = OnWindowInterceptTouch;
+                    using TouchDataSignal signal = new TouchDataSignal(Interop.ActorSignal.ActorInterceptTouchSignal(Layer.getCPtr(GetRootLayer())), false);
+                    signal?.Connect(rootLayerInterceptTouchDataCallback);
```

Comment:
I'm not familiar with this part, so I'm just curious about this line. Is there any possibilty that the signal is null?
In my opinion, the signal is just created so that it seems not possible to be null.

Reply:
This is the same as any other event handling logic. 
The signal null check is known as Mr dongsug-song's intention.
Isn't this code in case the native object is destroyed?

Reply:
Thank you for the explanation. :+1: